README.md

```diff
@@ -57,7 +48,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: this is O(n^2) because there are two nested .each loops.
```

Comment:
I think I might have confused some people since someone else mentioned this logic too. O(n * m) does not simplify to O(n^2). I just tend to confuse them sometimes. O(n*m) is when the two loops have different data sets that they are iterating over. O(n^2) is when the two loops iterate over the same data set.